CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Throw a proper exception when operating on a non-existing field with the dynamic API (#3292).
 * `DynamicRealmObject.setList` should only accept `RealmList<DynamicRealmObject>` (#3280).
 * `DynamicRealmObject.getX(fieldName)` now throws a proper exception instead of a native crash when called with a field name of the wrong type (#3294).
+* Fixed a problem that `Table` and schema cache never hits against proxy classes (#3315).
```

Comment:
There are two problems.

One is that as you said we have two different maps for cache.

And another is that we put only original model classes as a key and never put proxy classes as a key. This means that cache entry is never used and we always recreate the cache entry when we pass a proxy class as an argument of `getTable()`.

I only mentioned the latter one. Should I mention both?